src/registrar/templates/home.html

```diff
@@ -9,189 +9,48 @@
-            <button class="usa-button usa-button--unstyled margin-right-2 domain-requests__reset-button display-none" type="button">
-              Reset
-            </button>
-            <label class="usa-sr-only" for="domain-requests__search-field">Search</label>
-            <input
-              class="usa-input"
-              id="domain-requests__search-field"
-              type="search"
-              name="search"
-              placeholder="Search by domain name"
-            />
-            <button class="usa-button" type="submit" id="domain-requests__search-field-submit">
-              <img
-                src="{% static 'img/usa-icons-bg/search--white.svg' %}"
-                class="usa-search__submit-icon"
-                alt="Search"
-              />
-            </button>
-          </form>
-        </section>
-      </div>
-    </div>
-    <div class="domain-requests__table-wrapper display-none">
-      <table class="usa-table usa-table--borderless usa-table--stacked dotgov-table dotgov-table--stacked domain-requests__table">
-        <caption class="sr-only">Your domain requests</caption>
-        <thead>
-          <tr>
-            <th data-sortable="requested_domain__name" scope="col" role="columnheader">Domain name</th>
-            <th data-sortable="submission_date" scope="col" role="columnheader">Date submitted</th>
-            <th data-sortable="status" scope="col" role="columnheader">Status</th>
-            <th scope="col" role="columnheader"><span class="usa-sr-only">Action</span></th>
-            <!-- AJAX will conditionally add a th for delete actions -->
-          </tr>
-        </thead>
-        <tbody id="domain-requests-tbody">
-          <!-- AJAX will populate this tbody -->
-        </tbody>
-      </table>
-      <div
-        class="usa-sr-only usa-table__announcement-region"
-        aria-live="polite"
-      ></div>
-    </div>
-    <div class="domain-requests__no-data display-none">
-      <p>You haven't requested any domains.</p>
-    </div>
-    <div class="domain-requests__no-search-results display-none">
-      <p>No results found for "<span class="domain-requests__search-term"></span>"</p>
-    </div> 
-  </section>
-  <nav aria-label="Pagination" class="usa-pagination flex-justify" id="domain-requests-pagination">
-    <span class="usa-pagination__counter text-base-dark padding-left-2 margin-bottom-1">
-      <!-- Count will be dynamically populated by JS -->
-    </span>
-    <ul class="usa-pagination__list">
-      <!-- Pagination links will be dynamically populated by JS -->
-    </ul>
-  </nav>
+    {% include "includes/domains_table.html" %}
+    {% include "includes/domain_requests_table.html" %}
 
-  {# Note: Reimplement this after MVP #}
-  <!--
-  <section class="section--outlined tablet:grid-col-11 desktop:grid-col-10">
-    <h2>Archived domains</h2>
-    <p>You don't have any archived domains</p>
-  </section>
-  -->
+    {# Note: Reimplement this after MVP #}
```

Comment:
we should just delete this code until we are ready to use it. There may not be an archived domains section until 2025 or 2026, seems odd for us to now keep this in.

Reply:
removed